public/static/locales/en-US/schedule.json

```diff
@@ -2,7 +2,12 @@
   "title": {
     "student": "Schedule",
     "studentWithClass": "M.{{number}} schedule",
-    "teacher": "Schedule"
+    "teacher": "Schedule",
+    "inaccurateNotice": {
+      "start": "This schedule may not be accurate. Please check your official schedule from the ",
+      "url": "school website",
+      "end": "."
+    }
```

Comment:
This should not be nested inside `title`. Although I have to admit that the translation file situation is a bit of a mess right now.